docs/implementation_plans/2026-02-22_agent_prototypes_learning_feedback.md

```
@@ -0,0 +1,913 @@
+  /// 4. Return the proposed new version for user preview/approval.
+  Future<ProposedTemplateVersion> evolve({
+    required AgentTemplateVersionEntity currentVersion,
+    required TemplatePerformanceMetrics metrics,
+    required UserFeedback feedback,
+  });
+}
+```
+
+The meta-prompt for evolution:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)